docs/examples/cloudauditlogssource/README.md

```diff
@@ -55,10 +55,7 @@ Create a GCP PubSub Topic:
 We will verify that the published event was sent by looking at the logs of the
 service that this CloudAuditLogsSource sinks to.
 
-1. We need to wait for the downstream pods to get started and receive our event,
```

Comment:
sorry, I explained it wrong. 
I meant to keep the paragraph, and add the sentence that was below... It will look something like:
```
1. We need to wait for the downstream pods to get started and receive our event,
   wait 60 seconds. You can check the status of the downstream pods with:
```

But you can discard it if you want, and keep it as it was before.